### ISSUE TITLE
x/tools/gopls: report if accepting a completion item will result in auto-import

### ISSUE THREAD

**Issue by xzbdmw** [golang/go#70975]
### gopls version

0.17.1

### go env

```shell
..
```

### What did you do?

"os" is not yet imported:
<img width="888" alt="image" src="https://github.com/user-attachments/assets/447f0444-ccd4-48ef-b999-2eaa53dd7a70" />

however, strings are already imported:
<img width="917" alt="image" src="https://github.com/user-attachments/assets/8b1a86f9-7311-4470-b525-8740a6a06415" />

### What did you see happen?

They looks the same.

### What did you expect to see?

Show a hint whether a package is going to be auto-imported, because some package names look similar, I want to use an already imported one, but find myself often accept a wrong name, therefor need to undo auto import.

In clangd, there is a little circle indicating this:
<img width="915" alt="image" src="https://github.com/user-attachments/assets/985723dd-3e34-4646-ac44-6658c9f72b2a" />

after accpeting `printf`, the circle disappears, because `stdio.h` is auto imported
<img width="874" alt="image" src="https://github.com/user-attachments/assets/8e0ce01c-fa24-45ac-94c4-a72cd4b2a3be" />

**Comment by findleyr** [comment]
Thanks. Coincidentally, I am working on unimported completion, so I will look into this.

Off the top of my head, I do not know what causes the cdot for the clang analyzer, but I can investigate. This is VS Code, right?

**Comment by xzbdmw** [comment]
Yes, the circle is part of the returned completion item, so every editor has the same looking
https://clangd.llvm.org/features#namespace-and-include-insertion

**Comment by h9jiang** [comment]
We have a commit merged for the label details so I was wondering whether we should close this. Turns out it is not exactly what this feature request is about but it is indeed a very good improvement to gopls's completion.

For this feature request, I do not think this is a standard feature that is provided by the LSP, as the completion item does not have such dedicated field indicating whether this completion item will result in import changes.

Purely a prefix `•` to `label = "•printf",`.

However, I think it might be a useful thing to do, as some of the completion item apply edits in place (where the user can see it explicitly), some of them include changes else where (import changes).

I will take this one and consider have it done for 0.25.0 (or if anyone is interested. + help wanted).
